memiavl/db.go

```diff
@@ -11,13 +11,15 @@ import (
 	"sync"
 	"time"
 
+	"github.com/alitto/pond"
 	"github.com/cosmos/iavl"
 	"github.com/tidwall/wal"
 )
 
 const (
-	DefaultSnapshotInterval = 1000
-	LockFileName            = "LOCK"
+	DefaultSnapshotInterval    = 1000
+	LockFileName               = "LOCK"
+	DefaultSnapshotWriterLimit = 32
```

Comment:
Shall we use a smaller default value, maybe just 4

Reply:
> Shall we use a smaller default value, maybe just 4

fixed